src/camera/Camera.js

```diff
@@ -282,15 +319,15 @@ exports = Class(Rect, function(supr) {
   };
 
   /**
-    * Keeps the actor completely in the view of the camera.
-    * @func Camera#fullyOn
-    * @type {cameraUpdateFunction}
-    */
+   * Keeps the actor completely in the view of the camera.
```

Comment:
`fullyOn` API feels weird. I think there are a lot of better names for this. Something like `blockActor`, `blockActorX` etc. Could also tie it in to the camera walls some how, indicating that this actor cannot pass through them. It might also make more sense to toggle on / off once, and on each camera update, the check is made.